tests/unit/conftest.py

```diff
@@ -197,6 +200,15 @@ def acoustic_newsletters_mapping(dbsession):
     return {r.source: r.destination for r in records}
 
 
+@pytest.fixture
+def test_factories(dbsession):
+    for factory in factories.__all__:
+        setattr(
+            attrgetter(f"{factory}._meta")(factories), "sqlalchemy_session", dbsession
+        )
+    return factories
```

Comment:
As referenced in the PR description, this is how I ended up adding the `dbsession` fixture to the factories. I don't love it.